<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IconChooser</class>
 <widget class="QFrame" name="IconChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Box</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="4" column="0">
    <widget class="IconView" name="iconsView" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="iconPasteLabel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>Click on empty place
to paste new icon
Right-click to cancel pasting</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="ItemIconLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Item Icon</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_10">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Hold &quot;Alt&quot; key to select icon</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="assignIconBtn">
     <property name="text">
      <string>Add custom Icon</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IconView</class>
   <extends>QWidget</extends>
   <header>iconview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
